### ISSUE TITLE
When Buildcraft is installed, the Distillery cannot make fuel.

### ISSUE THREAD

**Issue by Bluberry-Kat** [EpicSquid/GadgetryMachines#1]
Or, rather, JEI says it can use cool oil to make fuel, but whenever you try to put oil into the Distillery, it goes into the output tank (???)

Version 1.0.0 of Gadgetry
Version 2772 of Forge
Version 7.99.18 of Buildcraft

**Comment by TimGoll** [comment]
Maybe this one helps you: https://minecraft.curseforge.com/projects/what-fluid-fix

**Comment by Bluberry-Kat** [comment]
No. That mod when you have two different kinds of the same fluid, and forge doesn't bake the texture for one. 

My issue here is that, with Buildcraft installed, Gadgetry digs up Buildcraft oil, and Buildcraft oil cannot be processed in the Gadgetry refinery.

**Comment by Bluberry-Kat** [comment]
That being said, I just realized I left out a crucial detail for this issue:

I also have Tech Reborn, which also registers an oil fluid.

**Comment by TimGoll** [comment]
That was my thought exactly. But I didn't know that this mod is only for the visuals
